File: BOJ/boj2587.cpp
#include<stdio.h>

int main()
{
	int num[6];
	int i;
	int sum=0;
	int temp=0;

	for(i=1;i<=5;i++){
		scanf("%d",&num[i]);
		sum+=num[i];
	}

	for(i=1;i<=4;i++)
		for(int j=i+1;j<=5;j++)
			if(num[i]>num[j]){
				temp=num[j];
				num[j]=num[i];
				num[i]=temp;
			}

	printf("%d\n",sum/5);
	printf("%d",num[3]);




}

File: BOJ/boj2587.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		
		int[] num = new int[5];
		int sum=0;
		int tmp=0;
		
		Scanner scanner = new Scanner(System.in);
		
		for(int i=0;i<num.length;i++){
			num[i]= Integer.parseInt((scanner.nextLine().trim()));
			sum+=num[i];
		}
		
		for(int i=0;i<num.length-1;i++){
			for(int j=i+1;j<num.length;j++){
				
				if(num[i]>num[j]){
					tmp=num[i];
					num[i]=num[j];
					num[j]=tmp;
				}
			}	
		}
		
		
		System.out.printf("%d\n%d",(int)(sum/5),num[2]);
		
		scanner.close();
		scanner=null;
	}

}
